src/compas_fab/backends/ros/client.py

```diff
@@ -151,7 +151,6 @@ def __enter__(self):
 
     def __exit__(self, *args):
         self.planner.dispose_planner()
```

Comment:
Shouldn't we get rid of this line as well?
```suggestion
```

Reply:
oh, how did i miss that?